broadcaster/lib/broadcaster_web/router.ex

```diff
@@ -44,6 +44,12 @@ defmodule BroadcasterWeb.Router do
       post "/sse", MediaController, :sse
       post "/layer", MediaController, :layer
     end
+
+    scope "/admin" do
+      pipe_through :auth
+
+      delete "/chat/:id", PageController, :delete_chat_message
+    end
```

Comment:
I think we can merge this with already exisiting scoep "admin", can't we?

Reply:
The `admin` scope is piped through `:browser`, so we cannot use it for that. I could refactor the `admin` scope a bit, but I'm not sure if it's worth it.